README.md

```diff
@@ -1,26 +1,56 @@
 # spicetify community themes
 
-This is a collection of themes for [spicetify](https://github.com/khanhas/spicetify-cli), a command-line tool to customize Spotify; you can add your own theme simply by opening a Pull Requests (more info in the Contributions section).
+This is a collection of themes for [spicetify](https://github.com/khanhas/spicetify-cli), a command-line tool to customize Spotify. 
+
+You can add your own theme simply by opening a Pull Request (more info in the [Contributions section](#contributions)).
 
 ### **You can find a preview of all the themes in the [wiki](https://github.com/morpheusthewhite/spicetify-themes/wiki/Themes-preview).**
 
-**Note that these themes require you to have the old Spotify UI (<v1.1.56) and Spicetify <v2**
+## Notes:
+- **These themes require you to have the old Spotify UI (<v1.1.56) and [Spicetify <v2](https://github.com/khanhas/spicetify-cli/wiki/Installation#legacy-spotify).**
+- **To install Dribbblish and DribbblishDynamic, follow the instructions in their READMEs**.  
+- **Spotify ad-blocked version is not supported.**
 
 ## Installation and usage
 
 (If you use Arch Linux you can find this project on the [AUR](https://aur.archlinux.org/packages/spicetify-themes-git/))
 
-Once you cloned the repository you'll need to put the files into the Themes folder. This varies between operating systems. The example shows the `Themes` directory for Linux. For other operating systems, see the `Themes` folder location [here](https://github.com/khanhas/spicetify-cli/wiki/Customization#themes).
+1. Clone this repository. Make sure [git](https://git-scm.com/) is installed and run:
+
+```bash
+git clone https://github.com/morpheusthewhite/spicetify-themes.git
+```
```

Comment:
You probably need to indent this to make it part of the numbered element. Same for the other numbered elements

Reply:
Sorry this took me a while, I didn't realise markdown needed a 4 space indent for this to work :P

Reply:
Done in e581510